src/lib/radio/radio.spec.ts

```diff
@@ -80,6 +80,15 @@ describe('MdRadio', () => {
       }
     });
 
+    it('should set required to each individual radio when the group is requried', () => {
+      testComponent.isGroupRequired = true;
+      fixture.detectChanges();
+
+      for (let radio of radioInstances) {
+        expect(radio.required).toBe(true);
+      }
+    });
+
```

Comment:
There should also be a test that an individual radio button with `required = true` properly sets `required` to the underlying native `<input type="radio">`